nf_core/modules/update.py

```diff
@@ -433,12 +433,6 @@ class DiffEnum(enum.Enum):
             if not dry_run:
                 self.update_modules_json(modules_json, modules_repo.name, module, version)
 
```

Comment:
This change was introduced from a previous PR by mistake 
```suggestion

            # Don't save to a file, just iteratively update the variable
            else:
                modules_json = self.update_modules_json(
                    modules_json, modules_repo.name, module, version, write_file=False
                )
                
```